### PR TITLE
Add microglia morphometry

### PR DESCRIPTION
**Pull request by tischi**

### CODE REVIEW

File: sites.yml [hunk 1]
```diff
@@ -206,7 +206,14 @@ sites:
       [this forum post](https://forum.image.sc/t/bigvolumeviewer-tech-demo)
       for getting started.
     maintainers:
-      - "[Tobias Pietzsch](https://imagej.net/User:Pietzsch)"
+      - "  - name: "BigDataProcessor"
```
Comment (imagejan) [inline sites.yml:209]: There's something wrong here, you deleted part of the previous entry.

File: sites.yml [hunk 2]
```diff
@@ -218,7 +225,7 @@ sites:
       The last SNAPSHOTS are automatically uploaded daily.
     maintainers:
       - "[OME development team](https://www.openmicroscopy.org)"
-
+git@github.com:imagej/list-of-update-sites.git
```
Comment (imagejan) [inline sites.yml:228]: ???

File: sites.yml [hunk 3]
```diff
@@ -505,7 +540,14 @@ sites:
     description: >-
       DiversePathsJ implements a dynamic programming approach for the general
       task of finding diverse shortest paths between two end-points. It is not
-      linked to a specific biological problem and can be applied to a large
+      linked to a specific biological problem and can be applie  - name: "BigDataProcessor"
```
Comment (imagejan) [inline sites.yml:543]: Some information got deleted here.

File: sites.yml [hunk 4]
```diff
@@ -505,7 +540,14 @@ sites:
...
+    id: "BigDataProcessor"
+    url: "https://sites.imagej.net/BigDataProcessor/"
+    description: >-
+      [BigDataProcessor2](https://github.com/embl-cba/bigDataProcessor2#bigdataprocessor2)
+      is an Imglib2 and BigDataViewer based tool for inspection and processing of big (TB-sized) image data.
+    maintainers:
+      - "[Christian Tischer](mailto:christian.tischer@embl.de)"d to a large
```
Comment (imagejan) [inline sites.yml:550]: ??

**Review by imagejan — CHANGES_REQUESTED** [review]
This looks like wild copy-and-paste to me, with additions of the same information in several places. @tischi can you please make sure the .yml file stays intact?
(We should have checks in place on our continuous integration (GitHub Actions), but they're temporarily not working due to the move from Travis to GitHub (see https://github.com/imagej/list-of-update-sites/issues/91). Otherwise the CI would have caught this messed up yml syntax.)

**Comment by tischi** [comment]
This is very weird as I did not touch those regions at all...
I will check!

**Comment by tischi** [comment]
There is something screwed up. I'll delete this branch and make a new PR.

**Pull request closed without merging**